core/vm/contracts_suave.go

```diff
@@ -109,6 +112,28 @@ func (b *suaveRuntime) fetchBids(targetBlock uint64, namespace string) ([]types.
 	return bids, nil
 }
 
+func (b *suaveRuntime) randomUint() (*big.Int, error) {
```

Comment:
should the max return value here be u256 or u64?

Reply:
It should be u256 -- does golang's `big.Int` not go that high?

Reply:
`big.Int` is arbitrarily precision.